includes/class-wc-calypso-bridge-ecommerce-admin-menu.php

```diff
@@ -399,6 +400,48 @@ public function maybe_hide_payments_menu() {
 		}
 	}
 
+	/**
+	 * Hide the Customizer menu item when a block theme is used.
+	 *
+	 * Keep in mind that if there is custom css, a new menu item will appear `Additional CSS`
+	 * so there is no need to handle this case.
+	 *
+	 * @since x.x.x
+	 */
+	public function maybe_remove_customizer_menu() {
+
+		global $submenu;
+
+		if ( ! wp_is_block_theme() ) {
+			return;
+		}
+
+		if ( isset( $submenu['themes.php'] ) ) {
+			foreach ( $submenu['themes.php'] as $item ) {
+				if ( isset( $item[2] ) && strpos( $item[2], 'customize.php' ) !== false ) {
+					remove_submenu_page( 'themes.php', $item[2] );
```

Comment:
It's better to hide the menu instead of removing it. Take a look at this class's `hide_submenu_page` method.

Reply:
You can also break the loop, once its found.

Reply:
Makes sense! Thanks